### ISSUE TITLE
No Interact Button after logging in 

### ISSUE THREAD

**Issue by pkash35** [a16z-infra/ai-town#165]
Hi, we've been having an issue with finding the interact button after we log in with convex. We have the latest files as well.

**Comment by pkash35** [comment]
We found that when the isauthenticated function was commented out from the interactbutton.tsx file, the button showed up. We still couldn't interact with it though.

**Comment by yanmengxiang1** [comment]
same question, any progress?

**Comment by 61cygni** [comment]
Yeah, this is an issue with Clerk we've not gotten to the bottom of.
However, all the interaction code is still there. Ian submitted a patch,
https://github.com/a16z-infra/ai-town/compare/main...testing/interact-without-clerk
which pulls out Clerk. I verified that if you use that, you can interact.
For local testing that should be fine for now. However, I wouldn't make it
public without auth.
